src/test/java/uk/ac/ebi/eva/pipeline/runner/EvaPipelineJobLauncherCommandLineRunnerTest.java

```diff
@@ -0,0 +1,229 @@
+    }
+
+    @Test
+    public void jobProvidedButNoParameters() throws JobExecutionException {
+        evaPipelineJobLauncherCommandLineRunner.setJobName(GENOTYPED_VCF_JOB);
+        evaPipelineJobLauncherCommandLineRunner.run("--" + SPRING_BATCH_JOB_NAME_PROPERTY + "=" + GENOTYPED_VCF_JOB);
+        assertThat(capture.toString(), containsString(NO_JOB_PARAMETERS_HAVE_BEEN_PROVIDED));
+    }
+
+    @Test
+    public void genotypedVcfJobTest() throws JobExecutionException, IOException, URISyntaxException,
+            ClassNotFoundException, StorageManagerException, InstantiationException, IllegalAccessException {
+        String databaseName = mongoRule.getRandomTemporaryDatabaseName();
+        File inputFile = GenotypedVcfJobTestUtils.getInputFile();
+        String outputDirStats = temporaryFolderRule.newFolder().getAbsolutePath();
+        String outputDirAnnotation = temporaryFolderRule.newFolder().getAbsolutePath();
+
+        File variantsStatsFile = GenotypedVcfJobTestUtils.getVariantsStatsFile(outputDirStats);
+        File sourceStatsFile = GenotypedVcfJobTestUtils.getSourceStatsFile(outputDirStats);
+
+        File vepInputFile = GenotypedVcfJobTestUtils.getVepInputFile(outputDirAnnotation);
+        File vepOutputFile = GenotypedVcfJobTestUtils.getVepOutputFile(outputDirAnnotation);
+
+        File fasta = temporaryFolderRule.newFile();
+
+        int lastJobCount = jobExplorer.getJobInstanceCount(GENOTYPED_VCF_JOB);
+
+        evaPipelineJobLauncherCommandLineRunner.setJobName(GENOTYPED_VCF_JOB);
+        evaPipelineJobLauncherCommandLineRunner.run(new EvaCommandLineBuilder()
+                .inputVcf(inputFile.getAbsolutePath())
+                .inputVcfId(GenotypedVcfJobTestUtils.INPUT_VCF_ID)
+                .inputVcfAggregation("NONE")
+                .inputStudyName("small vcf")
+                .inputStudyId(GenotypedVcfJobTestUtils.INPUT_STUDY_ID)
+                .inputStudyType("COLLECTION")
+                .outputDirAnnotation(outputDirAnnotation)
+                .outputDirStatistics(outputDirStats)
+                .databaseName(databaseName)
+                .appVepPath(GenotypedVcfJobTestUtils.getMockVep().getPath())
+                .vepCachePath("")
+                .vepCacheSpecies("human")
+                .vepCacheVersion("1")
+                .vepNumForks("1")
+                .inputFasta(fasta.getAbsolutePath())
+                .configDbReadPreference("secondary")
+                .dbCollectionsVariantsName("variants")
+                .dbCollectionsFilesName("files")
+                .dbCollectionsFeaturesName("features")
+                .dbCollectionsStatisticsName("populationStatistics").build()
+        );
+
+        assertEquals(EvaPipelineJobLauncherCommandLineRunner.EXIT_WITHOUT_ERRORS, evaPipelineJobLauncherCommandLineRunner.getExitCode());
+
+        assertFalse(jobExplorer.getJobInstances(GENOTYPED_VCF_JOB, lastJobCount, 1).isEmpty());
```

Comment:
the second parameter to getJobInstances is the index of the first job to return, if I'm not mistaken, we are passing here `getJobInstances(..., 1, 1)` and it should be `getJobInstances(..., 0, 1)` to get just the first job instance